source/_integrations/plugwise.markdown

```diff
@@ -236,33 +240,42 @@ script:
           preset_mode: asleep
 ```
 
-### Supported devices
+## Supported devices
+
+### Adam
+
+A complete zone control system also known as Adam HA, supporting:
+
+- On/Off, OpenTherm or Loria/Thermastage heating, and cooling support.
+- Running firmwares v3.x or v2.3.
+- Additional devices:
+  - Zone thermostats such as Lisa or Anna (see warning below on Anna),
+  - A temperature sensor, Jip,
+  - Valve controllers called Floor or Tom,
+  - An under-floor heating controller Koen (note: a Koen always comes with a Plug, which is the active part),
+  - Smart switches, either Plug or Aqara Smart Plug.
+
+### Anna
 
-The current implementation of the Python module (Plugwise-Smile) includes:
+A smart thermostat, supporting:
 
-Adam (zone_control) with On/Off, OpenTherm, and Loria/Thermastage heating and cooling support:
+- On/Off, OpenTherm heating, and Elga or Loria/Thermastage with heating and cooling support.
+- Running firmware v4.x, v3.x or v1.x.
 
- - v3.x
- - v2.3
+### P1 (DSMR)
 
- - Devices supported are Anna, Lisa, Jip, Floor, Tom, Plug, Aqara Smart Plug, and Koen (a Koen always comes with a Plug, the active part)
+A smart meter monitor for single or multiphase P1 and gas usage monitoring with the P1 running firmware v4.x, v3.x or v2.x.
```

Comment:
A P1 smart meter monitor for the single- or multi-phase grid power connection to your home ...